internal/server/proto/server.proto

```diff
@@ -1787,6 +1801,18 @@ message Job {
     Job.DataSource.Ref new_ref = 3;
   }
 
+  // InitOp triggers an init action for a project (the equivalent of
+  // `waypoint init`). The job will fail if there is no data source
+  // configured for the project.
+  message InitOp {
```

Comment:
I would comment and say this is safe to call multiple times (idempotent).

Reply:
Addressed in 19813f0489b2794f721763ef573c428544abb2dc